examples/llama-70b-benchmarking.yaml

```diff
@@ -0,0 +1,54 @@
+training:
+  train_iters: 100
+  num_workers: 8
+  logs:
+    interval: 10
+  validation:
+    iterations: null
+  test_iters: 0
+batch:
+  sequence_length: 8192
+  micro_batch_size: 1
+  batch_size: 32
+data:
+  format: random
+  split: [1, 0, 0]
+optimizer:
+  learning_rate:
+    base: 1.0e-05
+    decay_style: constant
+    warmup_iterations: 0
+  weight_decay: 0.1
+  beta_1: 0.9
+  beta_2: 0.95
+model:
+  base_model:
+    transformer:
+      normalization:
```

Comment:
please add `add_linear_biases: false`, because Llama has `attention_bias: false`.

Reply:
This is already in line 35